content/Assets/Styles/components/nav/_social.scss

```diff
@@ -0,0 +1,9 @@
+/* ==========================================================================
+   #NAV/SOCIAL
+   ========================================================================== */
+
+.nav-social {
```

Comment:
For BEM purposes I think this should be `.nav--social` (and same in the markup) - it's a modified version of an existing block (nav)

Reply:
Not sure how I missed this. Especially as I'd have typed it out at least twice in separate files. Good spot - I've pushed changes.